### PR TITLE
Lidar in Download Feature

### PR DESCRIPTION
**Pull request by ThorTheDestroyer**
- [x] First Draft @ThorTheDestroyer 
- [x] Review text in markdown @KCMack @pwblanton
- [x] Add images, formatting edits @christoball
- [ ] Review built dev version @KCMack @pwblanton 
- [ ] Merge Pull Request and Publish @christoball

### CODE REVIEW

File: content/markdown/news/2017-06-06_Lidar-data-on-website.MD [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
+The Texas Natural Resources Information System (TNRIS) Strategic Mapping team is excited to announce that all Texas Lidar data currently in our inventory is now available to be directly downloaded through TNRIS website section [Data Download](https://tnris.org/data-download/#!/statewide). TNRIS has implemented a streamlined and cost-effective, cloud-based infrastructure repository for all datasets which allows our data to be accessed directly through the download service. As with all downloadable data sets, users will be able to use an interactive map to “drill down” and to select the desired areas of interest.
```
Comment (jseppi) [inline content/markdown/news/2017-06-06_Lidar-data-on-website.MD]: whoa! 🎉

File: content/markdown/news/2017-06-06_Lidar-data-on-website.MD [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Lidar Data now available for download
+
+The Texas Natural Resources Information System (TNRIS) Strategic Mapping team is excited to announce that all Texas Lidar data currently in our inventory is now available to be directly downloaded through TNRIS website section [Data Download](https://tnris.org/data-download/#!/statewide). TNRIS has implemented a streamlined and cost-effective, cloud-based infrastructure repository for all datasets which allows our data to be accessed directly through the download service. As with all downloadable data sets, users will be able to use an interactive map to “drill down” and to select the desired areas of interest.
```
Comment (DavidHickman) [inline content/markdown/news/2017-06-06_Lidar-data-on-website.MD]: :confetti_ball:

**Comment by christoball** [comment]
Ready to review dev version:
https://deploy.tnris.org/api/v1/preview/6e31dd71f345ced5fc95caab7385876a5b10cddb/news/2017-06-12/tnris-lidar-data-now-available-download/

@ThorTheDestroyer @KCMack @pwblanton

**Comment by pwblanton** [comment]
It looks great, Chris!

**Pull request merged**